.github/workflows/ci.yml

```diff
@@ -91,14 +80,21 @@ jobs:
             examples/luvit/**/*.lua
             examples/neovim/**/*.lua
             examples/xmake/**/*.lua
-            # FIXME: these fail to parse on Windows
-            !examples/lua/testes/api.lua
-            !examples/lua/testes/calls.lua
-            !examples/lua/testes/db.lua
+            # FIXME: these fail to parse
+            !examples/lua/testes/all.lua
+            !examples/lua/testes/closure.lua
+            !examples/lua/testes/code.lua
+            !examples/lua/testes/files.lua
+            !examples/lua/testes/goto.lua
+            !examples/lua/testes/literals.lua
+            !examples/lua/testes/locals.lua
+            !examples/lua/testes/math.lua
+            !examples/lua/testes/nextvar.lua
+            !examples/lua/testes/pm.lua
+            !examples/lua/testes/strings.lua
+            !examples/lua/testes/utf8.lua
```

Comment:
@MunifTanjim all these files have parser errors on `main` as well on the line
```lua
global * <const>
```

Reply:
🙃 https://github.com/lua/lua/commit/be8120906304a8658fab998587b969e0e42f5650

Added new syntax. And it's still unreleased.

Can we pin the commit to `e05590591410a5e007a1e3f1691f6c1cf9d8fe45` for that repo?